src/LanguageServer/Impl/Implementation/Server.WorkspaceSymbols.cs

```diff
@@ -14,9 +14,11 @@
 // permissions and limitations under the License.
```

Comment:
Not related to anything you've done per se, but this might be a good opportunity to rename this file to `Server.Symbols.cs`, since this implements all of the symbol calls (not just workspace symbols). Just a minor thought since it bugged me in the old codebase.